doctr/models/predictor/pytorch.py

```diff
@@ -25,6 +27,9 @@ class OCRPredictor(nn.Module, _OCRPredictor):
     Args:
         det_predictor: detection module
         reco_predictor: recognition module
+        straighten_pages: if True, evaluates the page general orientation based on the median of each line orientation.
+            Then, rotates page before using Detection and Recognition Predictors. Then rotates page back to original
```

Comment:
"before using..." --> "before passing it to the deep learning modules"

Reply:
ok done